### ISSUE TITLE
Extension-less Python files not picked up

### ISSUE THREAD

**Issue by ijl** [MagicStack/MagicPython#25]
Bit of a papercut, but...

If I have `example`, not no `.py` extension, as:

```
#!/usr/bin/env python

if __name__ == '__main__':
    print('python')
```

Then in Sublime Text 3, MagicPython isn't automatically selected. Without MagicPython enabled and Python disabled, the stock Python package is selected.

**Comment by vpetrovykh** [comment]
Thanks for reporting this. It will be fixed in the next release (soon).

**Comment by 1st1** [comment]
Should be fixed in `v0.4.43`

**Comment by ijl** [comment]
Thanks.
